src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -1123,7 +1114,15 @@ public static ISet<string> BuildDisabledSkimmersSet(TContext context, IEnumerabl
                 }
             }
 
-            return disabledSkimmers;
+            if (disabledSkimmers.Count == skimmers.Count())
```

Comment:
>Count() [](http://example.com/codeflow?start=51&length=7)

nit: since this is a sortedSet, you could use .Count instead

Reply:
I must have done something wrong with Git because I did not make this change.

Reply:
Initially I was trying to work with the repo directly but I wasn't allowed to push to a new branch.  So I created the fork and copied the files over.  They must not have been from the same commit so when I copied the files, this change crept in and I didn't notice it.